crates/auto-hash-map/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+pub mod map;
+pub mod set;
+
+pub use map::AutoMap;
+pub use set::AutoSet;
+
+/// Maximum size of list variant. Must convert to HashMap when bigger.
+pub const MAX_LIST_SIZE: usize = 32;
```

Comment:
Is this a magic number, or have you profiled and found that after 32, it is not worth it anymore to use list?

Reply:
Currently it's a guess, but we need to figure out a good number from benchmarking.